src/Employment/WorkContract_v0.1.py

```diff
@@ -0,0 +1,339 @@
+        examples=["Hotel guest reception"],
+    )
+    work_conditions: Optional[str] = Field(
+        None,
+        title="Work Conditions",
+        max_length=400,
+        description="The special conditions regarding the work duties, e.g. work for another party than the employer",
+        examples=["The work duties done for Lapland Hotels"],
+    )
+    industry: Optional[str] = Field(
+        None,
+        title="Industry",
+        max_length=10,
+        description="The work industry based on the Statistical classification of economic activities in the European "
+        "Community, abbreviated as NACE.",
+        examples=["79.1"],
+    )
+    locations: List[str] = Field(
+        None,
+        title="Locations",
+        description="The region in Finland where the work is done",
+        examples=["Rovaniemi, Lapland"],
+        max_length=250,
+    )
+    working_hours: float = Field(
+        ...,
+        title="Working Hours",
+        description="The number of work hours per week",
+        examples=[30],
+    )
+    collective_agreement: Optional[str] = Field(
+        None,
+        title="Collective Agreement",
+        max_length=250,
+        description="The name of the applicable collective agreement",
+        examples=["The collective agreement for Employment industry (HELA)"],
+    )
+    overtime_rules: str = Field(
+        ...,
+        title="Overtime Rules",
+        max_length=400,
+        description="The terms for the overtime work and compensation",
+        examples=["For overtime work you receive 25 € per overtime hour"],
+    )
+    probation: str = Field(
+        ...,
+        title="Probation",
+        max_length=250,
+        description="The terms for the period of conditional employment",
+        examples=[
+            "The probationary period lasts 2 months from the employment start date"
+        ],
+    )
+
+
+class PaymentGrounds(str, Enum):
+    MONTHLY = "monthly"
+    HOURLY = "hourly"
+
+
+class Compensation(CamelCaseModel):
+    payment_grounds: PaymentGrounds = Field(
+        ...,
+        title="Payment Grounds",
+        description="The grounds for paying the salary",
+        examples=[PaymentGrounds.MONTHLY, PaymentGrounds.HOURLY],
+    )
+    salary: float = Field(
+        ...,
+        title="Salary",
+        description="The amount of salary in euros",
+        examples=[2500],
+    )
+    bonuses: Optional[str] = Field(
+        None,
+        title="Bonuses",
+        description="The details of the salary bonuses",
+        examples=["No additional bonuses"],
+        max_length=250,
+    )
+    payment_time: str = Field(
+        ...,
+        title="Payment Time",
+        description="The conditions for paying the salary",
+        examples=["The 15th of every month"],
+        max_length=250,
+    )
+
+
+class BenefitType(str, Enum):
+    PART_OF_SALARY = "partOfSalary"
+    ADDITIONAL_TO_SALARY = "additionToSalary"
```

Comment:
Sorry, checked with Henna these and it was an error these were camelCased in the spreadsheet. They're now snake_cased in the spreadsheet, need to be reverted to that here as well.